readthedocs/proxito/views/serve.py

```diff
@@ -912,14 +912,15 @@ def changefreqs_generator():
 
             if project.translations.exists():
                 for translation in project.translations.all():
-                    translation_versions = (
```

Comment:
We were fetching all versions from the translation, when we just need to get a version matching the same slug as the current version.